### PR TITLE
Additions to Nightscout Remote Commands Documentation

### PR DESCRIPTION
**Pull request by dnzxy**
Several Loopers in the FB groups have had struggle around setting up Nightscout Remote Commands and creating a key that enables Apple Push Notification Service (APNS) for this key. This PR adds more specific instructions and adds a link to the specific section on the Apple Developer portal.

* Added link to Apple Dev Certificates, Indentifiers & Profiles keys section
* Fixed indentation issue that caused wrong list numbering

@ebouchut please let me know if / what specific terminology you'd like me to wrap in `<code>` tags to prevent auto-translation. I'll adjust the PR accordingly :)

### CODE REVIEW

File: docs/nightscout/remote-overrides.md [hunk 1]
```diff
@@ -70,34 +70,35 @@ Error messages if Looper's phone is not configured correctly:
 
 ## Step 2: Apple Push Notifications
 
-The next part of this will help your Loop app give permissions to your Nightscout site to remotely interact with it. (Reminder - this only works with the paid Apple Developer ID.)
+The next part of this will help your Loop app give permissions to your Nightscout site to remotely interact with it. To enable this, you need to create a key and grant it access to the Apple Push Notification Service (APNS). *(Reminder - this only works with the paid Apple Developer ID.)* 
 
-1. Login to your [Apple developer account](https://developer.apple.com/account/) with the Apple ID associated with your developer team that you used to sign your Loop app.
-2. Click on "Certificates, Indentifiers & Profiles" and then, on the next page, click on "Keys" (located on the left-hand column). Click on the blue "Create a new key" **OR** the &plus; icon to add a new key.
+1. To get started, go to the "Keys" section under Apple Developer's [Certificates, Identifiers & Profiles](https://developer.apple.com/account/resources/authkeys/list) and login with the Apple ID associated with your developer team that you used to sign your Loop app.
```
Comment (ebouchut) [inline docs/nightscout/remote-overrides.md:75]: I also either italicize or surround **brand/project/product names** with `<span translate="no">...</span>`  
> that you used to sign your `*Loop*` app

File: docs/nightscout/remote-overrides.md [hunk 2]
```diff
@@ -70,34 +70,35 @@ Error messages if Looper's phone is not configured correctly:
...
 
-![img/apns-add-key.png](img/apns-add-key.png)
+2. If not already open in your browser (compare with below screenshot), click on "Keys" (located on the left-hand column). Click on the blue "Create a new key" **OR** the &plus; icon to add a new key.
 
-3. In the form that appears, click the checkbox for enabling "Apple Push Notifications service (APNs)" and enter a name for the key such as "Nightscout". Then click the "Continue" button in the upper right of the screen.
+    ![img/apns-add-key.png](img/apns-add-key.png)
 
-![img/key-apns.png](img/key-apns.png)
+3. In the form that appears, do the following:
+    - Click the checkbox for enabling "**Apple Push Notifications service (APNs)**"
```
Comment (ebouchut) [inline docs/nightscout/remote-overrides.md:82]: `` **`Apple Push Notifications service (APNs)`** ``

File: docs/nightscout/remote-overrides.md [hunk 3]
```diff
@@ -70,34 +70,35 @@ Error messages if Looper's phone is not configured correctly:
...
-![img/apns-add-key.png](img/apns-add-key.png)
+2. If not already open in your browser (compare with below screenshot), click on "Keys" (located on the left-hand column). Click on the blue "Create a new key" **OR** the &plus; icon to add a new key.
 
-3. In the form that appears, click the checkbox for enabling "Apple Push Notifications service (APNs)" and enter a name for the key such as "Nightscout". Then click the "Continue" button in the upper right of the screen.
+    ![img/apns-add-key.png](img/apns-add-key.png)
 
-![img/key-apns.png](img/key-apns.png)
+3. In the form that appears, do the following:
+    - Click the checkbox for enabling "**Apple Push Notifications service (APNs)**" 
+    - Enter a name for the key such as "Nightscout" (you can name it however you want, just make sure you know what the key is for by the name you choose).
```
Comment (ebouchut) [inline docs/nightscout/remote-overrides.md:83]: `` `Nighscout` ``

File: docs/nightscout/remote-overrides.md [hunk 4]
```diff
@@ -70,34 +70,35 @@ Error messages if Looper's phone is not configured correctly:
...
+    - Then click the "Continue" button in the upper right of the screen.
```
Comment (ebouchut) [inline docs/nightscout/remote-overrides.md:84]: `` `Continue` ``

File: docs/nightscout/remote-overrides.md [hunk 5]
```diff
@@ -70,34 +70,35 @@ Error messages if Looper's phone is not configured correctly:
...
+   
+    ![img/key-apns.png](img/key-apns.png)
 
-4. In the screen that follows, click the blue "Register" button.
+4. In the screen that follows, click the blue "**Register**" button.
```
Comment (ebouchut) [inline docs/nightscout/remote-overrides.md:88]: `` **`Register`** ``

**Review by ebouchut — COMMENTED** [review]
LGTM, I suggested just a few tweaks here and there.

**Comment by ebouchut** [comment]
@dnzxy I made some additional changes to improve the automatic translation.
As I cannot push the branch to your repo, I pushed it to mine.
You can now hard-reset your local `gh-remote-commands-setup` branch with `ebouchut/gh-remote-commands-setup`, like so:

```shell
git stash
    
git remote add ebouchut git@github.com:ebouchut/loopdocs.git
git fetch ebouchut
    
git switch gh-remote-commands-setup
git reset --hard ebouchut/gh-remote-commands-setup
git push origin --force-with-lease gh-remote-commands-setup
```

**Comment by dnzxy** [comment]
@ebouchut your changes and additions are now pulled into the branch and the PR has been updated. I'll change the PR from draft to review.

**Comment by ebouchut** [comment]
Thx @dnzxy 
It is a big blob of changes but improves the automatic translation (at least in French).
Will update the README to add my newest findings regarding Google Translate.

**Pull request merged**